web/modules/custom/hoeringsportal_citizen_proposal/src/Helper/Helper.php

```diff
@@ -155,6 +159,27 @@ public function proposalPageAttachments(&$page): void {
     }
   }
 
+  /**
+   * Implements hook_entity_presave().
```

Comment:
This “Implements” declaration does not match the one in hoeringsportal_citizen_proposal.module.